test-suite/src/lib.rs

```diff
@@ -40,6 +40,14 @@ pub async fn run_command(command: impl AsRef<str>) {
     assert!(ecode.success());
 }
 
+/// Generates timestamp as the same as the node does.
```

Comment:
```suggestion
/// Generates timestamp the same as the node does.
```

Reply:
I think we are both wrong.
"Generates a timestamp in the same way as the node does." is correct.

Reply:
way better let's do that